integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/schema/CarbonAlterTableRenameCommand.scala

```diff
@@ -156,15 +156,20 @@ private[sql] case class CarbonAlterTableRenameCommand(
     } catch {
       case e: Exception =>
         LOGGER.error(e, "Rename table failed: " + e.getMessage)
-        if (carbonTable != null) {
-          AlterTableUtil
-            .revertRenameTableChanges(oldTableIdentifier,
-              newTableName,
-              carbonTable.getTablePath,
-              carbonTable.getCarbonTableIdentifier.getTableId,
-              timeStamp)(
-              sparkSession)
-          renameBadRecords(newTableName, oldTableName, oldDatabaseName)
+        try {
+          if (carbonTable != null) {
+            AlterTableUtil
+              .revertRenameTableChanges(oldTableIdentifier,
+                newTableName,
+                carbonTable.getTablePath,
+                carbonTable.getCarbonTableIdentifier.getTableId,
+                timeStamp)(
+                sparkSession)
+            renameBadRecords(newTableName, oldTableName, oldDatabaseName)
+          }
+        } catch {
+          case e: Exception =>
```

Comment:
use finally, without catch

Reply:
Here We need to release all old table acquired locks only in case of any exception.

Reply:
done